growth/too/models.py

```diff
@@ -945,6 +947,164 @@ class Observation(db.Model):
          comment='processed successfully?')
 
 
+class Observations(db.Model):
+    """Observation information, including the field ID, exposure time, and
+    filter."""
+
+    telescope = db.Column(
+        db.String,
+        db.ForeignKey(Telescope.telescope),
+        primary_key=True,
+        comment='Telescope')
+
+    dateobs = db.Column(
+        db.DateTime,
+        db.ForeignKey(Event.dateobs),
+        primary_key=True,
+        comment='UTC event timestamp')
+
+    localization_name = db.Column(
```

Comment:
In our database, plans are not associated with a particular localization, so neither should observations be.